google-storage/test/mockstorage.ts

```diff
@@ -0,0 +1,70 @@
+import { Observable } from "rxjs";
```

Comment:
The formatting in this file needs to be fixed.

I'd recommend running prettier through all the files in this project.